### ISSUE TITLE
MSI RX 550 Crash

### ISSUE THREAD

**Issue by juggernautsei** [nanopool/Claymore-Dual-Miner#114]
I read the READ.ME file and it states that the software does not support AMD 500 series.
Is that the reason that I am getting these errors? I have provided much reading material hoping it helps. 

Faulting application name: EthDcrMiner64.exe, version: 0.0.0.0, time stamp: 0x59a94db0
Faulting module name: ntdll.dll, version: 10.0.15063.0, time stamp: 0xb79b6ddb
Exception code: 0xc0000005
Fault offset: 0x00000000000a8e17
Faulting process id: 0x840
Faulting application start time: 0x01d339cb84095805
Faulting application path: C:\Users\Miner2\Downloads\NHML-1.8.1.3\bin_3rdparty\claymore_dual\EthDcrMiner64.exe
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report Id: 7d38d4b5-d234-42e8-bfff-2e7909a4df5e
Faulting package full name: 
Faulting package-relative application ID: 

[WERDD47.tmp.zip](https://github.com/nanopool/Claymore-Dual-Miner/files/1346353/WERDD47.tmp.zip)

**Comment by hesley** [comment]
friend I have RX 580 and miner ethereum goes normal.

**Comment by juggernautsei** [comment]
It was not a software issue it was a hardware issue.

**Comment by juggernautsei** [comment]
update to the hardware problem. 
ASRock should really put this in their documentation and in some video somewhere. Because all of the videos they have out there on building a 13GPU rig don't show this. 

1. Windows only support 8 of the same GPU of any one manufacturer.
2. All AMD cards have to be in the black slots and all the NVIDIA cards have to be in the white slots or vice verse.

**Comment by aenciso** [comment]
@juggernautsei point 1 is no the manual/website, point 2 definitively should be somewhere.
I've read that with latest windows creators update now you can have 13 nvidia gpus on windows and latest amd driver support more than 13 gpus on windows as well. I haven tried myself but I will at some point...

**Comment by juggernautsei** [comment]
I am writing you what is happening with me right now. I have 12 AMD cards and windows will only recognize 8 no more. I am running the latest AMD driver and the system is crashing and hanging like you would not believe. I have been backing down the system trying to get it stable. And I finally got it stable putting all 6 AMD cards in the black slots only since I don't have any NVIDIA cards.  I'll post a picture later.

**Comment by juggernautsei** [comment]
![20171101_123814](https://user-images.githubusercontent.com/834958/32285890-141afc8e-bf02-11e7-83cb-5e8066789ec3.jpg)
Here is my proof

**Comment by aenciso** [comment]
@juggernautsei what windows version do you have?

**Comment by juggernautsei** [comment]
I have 1703

**Comment by aenciso** [comment]
@juggernautsei I don't think it needs to be pro just make sure you have installed all latest windows updates.

**Comment by juggernautsei** [comment]
I don't read the information that you are sharing. On the AMD site for the driver, it says.

**Known Issues:**
A random system hang may be experienced after extended periods of use on system configurations using 12 GPU's for compute workloads.

This is what I was experiencing with my rig. And the windows creator has nothing to do with GPUs that I could find.

**Comment by Roaders** [comment]
> All AMD cards have to be in the black slots and all the NVIDIA cards have to be in the white slots or vice verse.

Is that what ASRock tech support told you? So you can't have 8 Nvidia and 4 AMD? That seems very odd. Is this just for windows?
I have definitely ran with a mix of Nvidia and Amd on white or black (4 AMD on one side, 4 Nvidia and 1 AMD on the other side) but perhaps this is why I was seeing some inconsistent results.

**Comment by juggernautsei** [comment]
@Roaders -  Here is what ASRock shared with me for troubleshooting
Thank you for contacting ASRock support.
1. Due to OS limitation, Windows cannot identify more than 8 same branded GPU, so user can only build hybrid system(A+N) under windows
2. For now only Linux can support 13 same branded GPU mining, there are 2 professional mining OS to choose from, both of which are easy to use and more importantly proven by the OS developer that is compatible with H110 Pro BTC+
3. Both Linux mining OS are paid software
4. We will frequently update our mining micro site, please visit here for the most updated software support info
http://www.asrock.com/microsite/H110ProBTC+/
 
Youtube video: https://www.youtube.com/watch?v=c2EDN7xyjZ4
                           https://www.youtube.com/watch?v=B6gB7GYkkiA
 
We have tested with 8 NVIDIA cards in the Windows environment, and it could be detected and worked properly in the system.
Hence, we would like to share some suggestions with the customer.
1. Please test with one VGA card on each PCIe slot to make sure all PCIe slots working individually.
2. Test all VGA card one by one to assure all VGA cards working individually.
3. Test make sure all PCIe x1 slots are connected to the power supply by PCIe x1 to x16 riser card.
4. Install VGA cards on black slots first, and install another two VGA cards on white one.

**Comment by juggernautsei** [comment]
@hesley @aenciso @Roaders 
I put all of my cards on the black connectors because I was desperate to get the system to work for more than a couple of hours. The rig will run for approximately 18hrs before locking up. Even with the new configuration of using only the black ports.

**Comment by juggernautsei** [comment]
Today the NiceHash switched my rigs to mining using sgminer and I am in the cryptonight pool. It is working well at this point. It did lock up after 18 hours of running. What I am thinking of doing is to have the PC reboot every 12 hours. That should keep it mining without locking up. 

I am writing this chronical to help someone else out there.

**Comment by juggernautsei** [comment]
Here is the latest with what is going on with my rig. My second rig that I have been trying to fix here is now working smoothly. It has been operational more than 24hrs. It is now mining Cryptonight with an increase in mBTC/day. Now it is time to build a third rig from all the leftover parts from rig 2. Of course, I am working on building a farm.

**Comment by juggernautsei** [comment]
I keep going round and round with trying to keep the second rig going but I found a cool tool within the Radeon Settings that is better than the MSI Afterburner. It allows you to do the same thing as the Afterburner.  I had MSI afterburner working and one day it stopped. Today I was desperate and looking for answers and within the Radeon tool, I found this.  Now, my rig is running again. 

![msi-settings2](https://user-images.githubusercontent.com/834958/32815588-cad85af4-c982-11e7-871b-7acbeb6d43aa.png)

**Comment by juggernautsei** [comment]
Back again and the rig just won't run more than 3 hours and then crashes repeatedly for 5hr. 

Any suggestions?

**Comment by juggernautsei** [comment]
This will be the end of my story. I have been mining Ethereum for the past month. I gave up windows altogether. Now, I am using simplemining.net. It is a Linux based OS. It took me a few times to figure out how to use it but it is a lifesaver for me. No more pulling my hair out at 2 AM because the windows drivers crashed again after 2hrs. Don't get me wrong the simple mining OS keeps track of the reboots but the system reboots and keeps going. That is the key. The keep going part. Now, I rarely think about my rig being online. I am moving on to my third rig now and will continue to use simple mining OS. Also, a side benefit is that I have doubled my income. I was using nice hash and making roughly $100/mo. Now on  dwarfpool.com, that amount is every two weeks. So, I received a win-win all the way around. Thank God!!
